TCMSApp/src/client/app/runs/run-execute.html

```diff
@@ -59,90 +59,212 @@
                     </div>
                 </div>
             </div>
-            <div class="text-right">
```

Comment:
> What have you changed in template? can you please describe? and also why such a big commented block have appeared?

before I cloned Natasha's projects I have made some changes to runs page template. And now I just try to include part of my template to cloned project and I also forgot to remove old code. I'll remove this comments